packages/v4/src/content/design-guidelines/usage-and-behavior/charts/charts.md

```diff
@@ -10,7 +10,7 @@ _Charts are only available in React_
 Use charts to visualize data in an application. The type of chart you use will depend on your use case and the type of data you need to display.
 
 ## Elements
-![Chart](./img/axis-labels.png)
+<img src=“./img/axis-labels.png” alt=“Chart with axis labels” width="713"/> 
```

Comment:
Use normal quotes instead of fancy Mac quotes (`"` vs `“` and `”`)

Reply:
Merp. Just fixed this! @redallen

Reply:
It doesn't look fixed for me. I'm still seeing `src=“./img/axis-labels.png” alt=“Chart with axis labels”`